### PR TITLE
{bio}[foss/2015b] BioKanga 3.4.5 (REVIEW)

### PR DESCRIPTION
**Pull request by nathanhaigh**
Easyconfigs for installing BioKanga 3.4.5 as requested in #2246

### CODE REVIEW

**Comment by nathanhaigh** [comment]
Jenkins: ok to test

**Comment by nathanhaigh** [comment]
yes

File: easybuild/easyconfigs/b/BioKanga/BioKanga-3.4.5-foss-2015b.eb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+homepage = 'http://sourceforge.net/projects/biokanga'
+description = """BioKanga is an integrated toolkit of high performance bioinformatics
+  subprocesses targeting the challenges of next generation sequencing analytics.
+  Kanga is an acronym standing for 'K-mer Adaptive Next Generation Aligner'."""
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+
+sources = ['buildbiokanga_%s.zip' % version.replace('.', '_')]
+
```
Comment (boegel) [inline easybuild/easyconfigs/b/BioKanga/BioKanga-3.4.5-foss-2015b.eb]: please drop this empty lines, we usually keep `sources` and `source_urls` close to each other

File: easybuild/easyconfigs/b/BioKanga/BioKanga-3.4.5-foss-2015b.eb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+source_urls = ['http://sourceforge.net/projects/biokanga/files/']
+
+dependencies = [('GSL', '2.1')]
+
+preconfigopts = "chmod +x configure && "
+
+sanity_check_paths = {
+                      'files': ["bin/biokanga"],
+                      'dirs': []
+                     }
```
Comment (boegel) [inline easybuild/easyconfigs/b/BioKanga/BioKanga-3.4.5-foss-2015b.eb]: please fix indent (indent with just 4 spaces):

```python
sanity_check_paths = {
    'files': ["bin/biokanga"],
    'dirs': []
}
```

**Comment by boegel** [comment]
@nathanhaigh: looks good, just a couple of small style remarks

**Comment by boegel** [comment]
Jenkins: ok to test

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/7b0516e7373e203e713c for a full test report.

**Comment by boegel** [comment]
@nathanhaigh looks great, going in, thanks!

For future PRs, please take a look at our guidelines for PR titles, which makes it a lot easier to see at a glance what the PR is contributing, see https://github.com/hpcugent/easybuild/wiki/Policy-for-easyconfig-pull-requests .

I updated the title myself for this PR, it will make it easier for me to compose the release notes in a couple of weeks.

**Pull request merged**